aws/resource_aws_codedeploy_app.go

```diff
@@ -26,6 +27,16 @@ func resourceAwsCodeDeployApp() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"compute_platform": &schema.Schema{
```

Comment:
There are two subtle, but important bugs with this attribute currently. 😄 
* The read function is not calling `d.Set("") which has two implications: Terraform cannot detect drift of the attribute and in absence of the `d.Set()` call with the value, Terraform core is silently passing the configuration value into the Terraform state (which is why the acceptance testing passes)
* This attribute cannot be updated in `UpdateApplication` and therefore needs to be set as `ForceNew: true` to force new resource creation

One other minor nitpick: (the rest of the file hasn't been updated yet, but) as of Go 1.7, the `&schema.Schema` is extraneous as its defined by `map[string]*schema.Schema` above so we prefer to remove them now :+1: